src/main/java/io/kontur/disasterninja/controller/LayerController.java

```diff
@@ -86,6 +90,39 @@ public List<LayerSummaryDto> getSummaries(@RequestBody
                 .collect(Collectors.toList());
     }
 
+    @Operation(tags = "Layers", summary = "Get list of global layers")
+    @ApiResponse(responseCode = "200", description = "Retrieved list of global layers", content = @Content(mediaType =
+            APPLICATION_JSON_VALUE, array = @ArraySchema(schema = @Schema(implementation = LayerSummaryDto.class))))
+    @PostMapping(path = PATH_SEARCH_GLOBAL, produces = APPLICATION_JSON_VALUE)
+    public List<LayerSummaryDto> getGlobalLayers() {
+        return layerService.getGlobalLayers(null)
```

Comment:
That's right. But in order to use lambda in `LayerService` while obtaining layers (to avoid code duplication) each method in providers should have the same signature. That's why parameters are used. But this could be discussed and refactored

Reply:
getGlobalLayers itself is not used in lambda.